app/src/main/java/org/xbmc/kore/jsonrpc/HostConnection.java

```diff
@@ -119,10 +127,6 @@
 	 * Socket used to communicate through TCP
 	 */
 	private Socket socket = null;
-	/**
-	 * Listener {@link Thread} that will be listening on the TCP socket
-	 */
-	private Thread listenerThread = null;
```

Comment:
I'll just point out that i liked having this thread separate, as this is a "special" thread that is always running (blocked on the TCP connection) so it might merit a special treatment.